code/modules/mining/machine_processing.dm

```diff
@@ -4,7 +4,7 @@
 
 /obj/machinery/mineral
 	speed_process = TRUE
-	init_process = FALSE
+	init_process = TRUE // Must be true after init_process fix
```

Comment:
This is a required change to prevent mineral machines from no longer processing. The current machines require `speed_process = TRUE` and `init_process = FALSE` due to an error that is being resolved.